certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py

```diff
@@ -100,7 +111,7 @@ def setUp(self):
         from certbot_dns_rfc2136._internal.dns_rfc2136 import _RFC2136Client
 
         self.rfc2136_client = _RFC2136Client(SERVER, PORT, NAME, SECRET, dns.tsig.HMAC_MD5,
-        TIMEOUT)
+        False, TIMEOUT)
```

Comment:
We're changing the public signature? Are we holding this PR over to a major version? Are DNS plugins not covered by our external interface commitments?

Reply:
We're in the `_internal` directory so I think we don't care.

Reply:
🤦 Good point.